encore/slurm_queue.py

```diff
@@ -14,23 +14,28 @@ def __init__(self, job_id, job_directory, config = None):
         else:
             self.config = dict()
 
-    def get_batch_headers(self, model_plan):
-        mem_per_cpu = model_plan.get("mem_per_cpu", 6500)
```

Comment:
As far as I can tell, `model_plan` will never have `mem_per_cpu`. Both this and `cores_per_job` were set to reference it (and both referenced `mem_per_cpu`).